Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1272,6 +1272,20 @@ CSSRhythmicSizingEnabled:
     WebCore:
       default: false
 
+CSSScopingStylesEnabled:
+  type: bool
+  status: preview
```

Comment:
`testable`? Seems a little early for `preview`.

Reply:
Agreed, we definitely do not want to enable this in STP atm